src/hl/dataset.rs

```diff
@@ -907,8 +907,12 @@ macro_rules! impl_builder {
             self.builder.$name($($var),*); self
         }
     };
-    ($plist:ident: $name:ident($($var:ident: $ty:ty),*)) => {
+    ($(#[cfg(feature = $feature:literal)])*
```

Comment:
Just wondering, wouldn't something like `#[cfg($tt:tt)]` work, so you wouldn't have to hard-code specific syntax here?

Reply:
I think I tried, but this was not as straight forward as I thought. My experience with macros is too limited to see what went wrong in this instance